examples/CANTestESP32_BuiltIn_AltPins/CANTestESP32_BuiltIn_AltPins.ino

```diff
@@ -1,77 +0,0 @@
-#include <esp32_can.h>
```

Comment:
I think it makes sense to remove this example, because pin mappings would be defined outside of `esp32_can`.
I think changing pins at runtime via `setCANPins` is a really cool feature, and could potentially support some interesting hardware designs.